pkgs/racket-doc/scribblings/guide/concurrency.scrbl

```diff
@@ -101,8 +101,9 @@ items then send the results to the main thread.
               (match (thread-receive)
                 [(list oper1 oper2 result-thread)
                  (thread-send result-thread
-                              (format "~a + ~a = ~a"
+                              (format "~a ~a ~a = ~a"
                                       oper1
+                                      (if (equal? operation +) '+ '-)
```

Comment:
What about using `(object-name operation)` ?

Reply:
That is a better solution! I'll update accordingly